From bf85d26f0326d4ea20429c29fd408703ee5bad54 Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Wed, 13 Sep 2017 15:53:02 +0200
Subject: [PATCH] spice-compile-warnings.m4: don't define FORITFY_SOURCES

* otherwise fails when the native gcc already defines
  FORTIFY_SOURCES (e.g. in gentoo).
  | In file included from ../../git/server/inputs_channel.c:19:0:
  | ../config.h:17:0: error: "_FORTIFY_SOURCE" redefined [-Werror]
  |  # define _FORTIFY_SOURCE 2

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 m4/spice-compile-warnings.m4 | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/m4/spice-compile-warnings.m4 b/m4/spice-compile-warnings.m4
index 959971a2..55352ad6 100644
--- a/m4/spice-compile-warnings.m4
+++ b/m4/spice-compile-warnings.m4
@@ -109,13 +109,6 @@ AC_DEFUN([SPICE_COMPILE_WARNINGS],[
     # This should be < 1024 really. pixman_utils is the blackspot
     # preventing lower usage
     gl_WARN_ADD([-Wframe-larger-than=20460])
-
-    # Use improved glibc headers
-    AH_VERBATIM([FORTIFY_SOURCE],
-    [/* Enable compile-time and run-time bounds-checking, and some warnings. */
-#if __OPTIMIZE__
-# define _FORTIFY_SOURCE 2
-#endif
 ])
 
     # Extra special flags
-- 
2.14.1

